packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -91,13 +93,13 @@ const DropdownMenuTrigger = React.forwardRef((props, forwardedRef) => {
         // only call handler if it's the left button (mousedown gets triggered by all mouse buttons)
         // but not when the control key is pressed (avoiding MacOS right click)
         if (event.button === 0 && event.ctrlKey === false) {
-          context.setOpen((prevOpen) => !prevOpen);
+          context.onOpenChange((prevOpen) => !prevOpen);
```

Comment:
Here you go, that's an example where we still want that form, and that doesn't real well to me.

Reply:
Yeah I should have changed this to be `onOpenToggle` but I'll await your feedback on this one https://github.com/radix-ui/primitives/pull/483#discussion_r582815779